### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by neyida**

### CODE REVIEW

File: tti/app/index.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+export default function Index() {
+  return (
+    <View
+      style={{
+        flex: 1,
+        justifyContent: "center",
+        alignItems: "center",
+      }}
+    >
```
Comment (trietmchau) [inline tti/app/index.tsx:14]: Change to nativewind

File: tti/app/login-componets/login.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        color: 'black',
+        fontWeight: 'bold',
+    },
+    subTitle: {
+        fontSize: 16,
+        fontFamily: 'sans-serif',
+        textAlign: 'center',
+        marginBottom: 20,
+        color: 'black',
+        lineHeight: 24,
```
Comment (trietmchau) [inline tti/app/login-componets/login.tsx]: Change to nativewind

File: tti/components/Collapsible.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  heading: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    gap: 6,
+  },
+  content: {
+    marginTop: 6,
+    marginLeft: 24,
+  },
+});
```
Comment (trietmchau) [inline tti/components/Collapsible.tsx:41]: Change to nativewind

**Review by trietmchau — COMMENTED** [review]
Looks good, just change to nativewind

File: tti/app/index.tsx (hunk 1 above)
Comment (neyida) [reply tti/app/index.tsx:14]: Changed.. Might need to install dependencies for it to render but Im pretty sure the other team worked to do that already.

**Pull request closed without merging**